.renvignore

```diff
@@ -2,4 +2,5 @@ README.Rmd
 pipeline/00-ingest.R
 pipeline/06-export.R
 pipeline/07-api.R
-reports/
\ No newline at end of file
+reports/
+cache/
```

Comment:
This is to make sure that renv doesn't scan the cache dir that packages actually get installed to within Docker.